src/plot/plot.jl

```diff
@@ -48,26 +53,75 @@ function clusters(
 
     clusters!(
         ax,
-        clusters=clusters,
-        cluster_sequence=cluster_sequence,
-        cluster_radius=cluster_radius,
-        nodes=nodes,
-        centers=centers,
-        labels=labels
+        clusters,
+        cluster_radius,
+        nodes,
+        centers,
+        labels
     )
 
     return fig, ax
 end
+function clusters(
+    ;
+    cluster_sequence::Union{DataFrame,Nothing}=nothing,
+    cluster_radius::Real=0,
```

Comment:
If you know the specific type, use that instead of a more generic type like `Real`.

```suggestion
    cluster_radius::Float64=0.0,
```

Reply:
I thought `Real` would allow flexibility for `Float` or `Int`

Reply:
Addressed by [b1e4b65](https://github.com/ryu-AIMS/HierarchicalRouting.jl/pull/80/commits/b1e4b6521c69a0a43c6642447c485a793e74e789)